randovania/interface_common/generator_frontend.py

```diff
@@ -69,7 +69,7 @@ def generate_layout(
                 "generation_trick_amount_intermediate": LayoutTrickLevel.INTERMEDIATE,
                 "generation_trick_amount_advanced": LayoutTrickLevel.ADVANCED,
                 "generation_trick_amount_expert": LayoutTrickLevel.EXPERT,
-                "generation_trick_amount_hypermode": LayoutTrickLevel.HYPERMODE,
+                "generation_trick_amount_ludicrous": LayoutTrickLevel.LUDICROUS,
```

Comment:
I think this messes up data, I guess sentry doesn't keep it for very long anyway. @Miepee ?

Reply:
Metrics are currently disabled on the sentry side. They have plans to bring them back later.